app/templates/components/forms/profile-page.hbs

```diff
@@ -0,0 +1,34 @@
+<form class="ui large form" autocomplete="off" {{action 'submit' on='submit'}}>
```

Comment:
Let this just be a `ui form`. large form doesn't go well with the rest of the site and overall UI